manual/manual.adoc

```diff
@@ -239,23 +242,26 @@ monsters with your bare hands. Ammo: None
 image:../sprites/csawa0.png[Ripsaw] |
 1 | Designed for cutting through wood, the ripsaw
 also works well as a melee weapon for cutting through flesh. Ammo: None
-| **Handgun** | 2 | Your starter weapon. A few shots will take down low-level
-monsters but it’s best to upgrade before taking on any tougher opponents.
+| **Handgun** +
+image:../sprites/pista0.png[Handgun] |
+2 | Your starter weapon. Its main purpose is to let you fight your way
+to a better weapon, and to hit shootable switches without wasting a second bullet.
 Ammo: Bullets
 | **Pump-action Shotgun** +
 image:../sprites/shota0.png[Pump-action Shotgun] |
-3 | The main “workhorse” weapon; fires seven pellets
-in a tight cluster and is effective over both short and long ranges against
-low- and medium-level monsters. Ammo: Shells
+3 | Shoots seven pellets in a fan pattern, letting you hit multiple
+targets or one big one. Ammo: Shells
 | **Double-barrelled Shotgun** +
 image:../sprites/sgn2a0.png[Double-barrelled Shotgun] |
-3 | Twice as powerful as the pump-action shotgun
-but less effective at long ranges; good at short range against crowds of
+3 | The simpler flash hiding system guarantees better
```

Comment:
I'm not sure what this means. Flash hiding system?

Reply:
This assumes #1085 will be merged. This combined with the current DBSG creates a situation where the DBSG has a bright orange flash entirely out of the muzzle, while the PASG has a much darker red flash that's also coming out of the barrel shroud holes for some reason. By adding this passing bit of rationalization here we can at least kinda harmonize both this visual difference and the fact that the PASG is so much weaker.

It's not a good explanation at all, but my hope is that the new-old DBSG gets in and the entire lore about the shotguns gets the rewrite I keep mentioning in the Discord.

Reply:
The new explanation is even flimsier but at least I'm not trying to pack multiple firearms concepts into 3 words I can't even begin to figure out how to translate without DeepL or Google. The sort of person who'd notice this new explanation is bullshit will probably know the origins of the bizarre game mechanics the writer was stuck with trying to explain; the sort of person who doesn't notice or care should be fine with the repeated "strength" words giving the proper vibe.

Reply:
#1085 now includes both shotguns so I *could* introduce the warp gun theory into here, but tbh the wording that's in the PR now is vague enough to include even that, so I'm leaving it alone.